harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/fragment/FragmentHarvester.java

```diff
@@ -351,19 +362,25 @@ private void createOrUpdateSubtemplate(Element fragment) throws Exception {
 
         String schema = fragment.getAttributeValue("schema");
 
+        String title = fragment.getAttributeValue("title");
```

Comment:
Yep ok. To make distinction between contacts we are also using groups. Eg. at Ifremer we have more than 10k contact imported from LDAP and some are categorized by group EDMO, EDMERP, ... and you can use filter attribute to configure the editor to use a subset of the directory content (https://github.com/geonetwork/core-geonetwork/blob/develop/schemas/iso19139/src/main/plugin/iso19139/layout/config-editor.xml#L163)


```

          <action type="add"
                  name="edmerpName"
                  addDirective="data-gn-directory-entry-selector"
                  or="pointOfContact"
                  in="/mdb:MD_Metadata/mdb:identificationInfo/*">
            <directiveAttributes
                    data-template-add-action="false"
                    data-search-action="false"
                    data-popup-action="true"
                    data-template-type="contact"
                    data-filter='{"_groupPublished": "CONTACTS_EDMERP"}'
                    data-variables="gmd:role/gmd:CI_RoleCode/@codeListValue~edmerp"
                    data-transformation="contact-from-iso19139-to-iso19115-3"/>
          </action>
```